perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
Ok - that sounds like it's probably the best option for now. I still wish there were a better solution - is what you are doing going to make running cassandra_ycsb work? Or will any benchmark that has a non juju component not be able to run with any juju packages (since we can't call relate in that instance)? Is there a recommended way to deploy only part of a solution using jujus (just the cassandra cluster but not the clients)?